src/hooks.ts

```diff
@@ -1,4 +1,5 @@
-import { TypedUseSelectorHook, useSelector } from "react-redux";
-import type { AppState } from "./types";
+import { TypedUseSelectorHook, useSelector, useDispatch } from "react-redux";
+import type { AppDispatch, AppState } from "./types";
 
 export const useAppSelector: TypedUseSelectorHook<AppState> = useSelector;
+export const useAppDispatch = () => useDispatch<AppDispatch>();
```

Comment:
https://redux-toolkit.js.org/usage/usage-with-typescript#getting-the-dispatch-type